modules/gridss/1.0/gridss.smk

```diff
@@ -0,0 +1,501 @@
+    log: log = CFG["logs"]["gripss"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/gripss.log"
+    resources: 
+        **CFG["resources"]["gripss"]
+    params:
+        pon_dir = CFG["references"]["pon_dir"],
+        alt_build = lambda w: {
+            "grch37": "hg19", 
+            "hs37d5": "hg19", 
+            "hg38": "hg38"}[w.genome_build], 
+        opts = CFG["options"]["gripss"], 
+        jvmheap = lambda wildcards, resources: int(resources.mem_mb * 0.8)
+    conda: 
+        CFG["conda_envs"]["gripss"]
+    threads: 
+        CFG["threads"]["gripss"]
+    shell: 
+        op.as_one_line(""" 
+        gripss -Xms4G -Xmx{params.jvmheap}m 
+        -ref_genome {input.fasta} 
+        -breakend_pon {params.pon_dir}/gridss_pon_single_breakend.{params.alt_build}.bed 
+        -breakpoint_pon {params.pon_dir}/gridss_pon_breakpoint.{params.alt_build}.bedpe 
+        -breakpoint_hotspot {params.pon_dir}/KnownFusionPairs.{params.alt_build}.bedpe 
+        -input_vcf {input.vcf} 
+        -output_vcf {output.vcf} 
+        -tumor {wildcards.tumour_id} 
+        -reference {wildcards.normal_id}  
+        {params.opts} 
+        2>&1 | tee -a {log} 
+        """)
+    
+rule _gridss_filter_gripss: 
+    input: 
+        vcf = str(rules._gridss_run_gripss.output.vcf)
+    output: 
+        vcf = CFG["dirs"]["gripss"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/gridss_somatic_filtered.vcf.gz", 
+        tbi = CFG["dirs"]["gripss"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/gridss_somatic_filtered.vcf.gz.tbi"
+    conda: 
+        CFG["conda_envs"]["bcftools"]
+    shell: 
+        op.as_one_line("""
+        zcat {input.vcf} | 
+            awk '$7 == "PASS" || $1 ~ /^#/ ' | 
+            bcftools view -Oz -o {output.vcf} && 
+        tabix {output.vcf}
+        """)
+
+rule _gridss_gripss_to_bedpe: 
+    input: 
+        vcf = str(rules._gridss_filter_gripss.output.vcf)
+    output: 
+        bedpe = CFG["dirs"]["gripss"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/gridss_somatic_filtered.bedpe"
+    conda: 
+        CFG["conda_envs"]["svtools"]
+    shell: 
+        op.as_one_line("""
+        zcat {input.vcf} | 
+            awk '$1 ~ /^#/ || $5 ~ /:/' | 
+            svtools vcftobedpe -o {output.bedpe}
+        """)
+     
+
+# Symlink the final output files into the module results directory (under '99-outputs/')
+rule _gridss_output_viral_vcf:
+    input:
+        vcf = str(rules._gridss_unpaired_filter.output.vcf), 
+        tbi = str(rules._gridss_unpaired_filter.output.tbi), 
+        bedpe = str(rules._gridss_unpaired_to_bedpe.output.bedpe)
+    output:
+        vcf = CFG["dirs"]["outputs"] + "vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.gridss_viral_annotation_filtered.vcf.gz", 
```

Comment:
As an outsider the name of this file sounds like it's not SVs but rather it's just viral integration site predictions. Presumably it's both, right?